index.html

```diff
@@ -0,0 +1,754 @@
+</head>
+
+<body>
+
+    <header>
+        <nav>
+            <div class="justFlex">
+                <div class="logo3">
+                    <img src="assets/images/logo3.PNG" alt="logo3">
+                </div>
+                <div class="screenMenu">
+                    <div class="menu flex-spaceBetween">
+                        <ul class="menu-item">
+                            <li><a href="#">News</a></li>
+                            <li><a href="#">Events</a></li>
+                            <li><a href="#">Business</a></li>
+                            <li><a href="#">AMAs</a></li>
+                            <li><a href="#">Spaces</a></li>
+                            <li><a href="#">Terms & Conditions</a></li>
+                        </ul>
+                        <ul class="menu-item">
+                            <li><a href="#">About</a></li>
+                            <li><a href="#">Advertise</a></li>
+                            <li><a href="#">Jobs</a></li>
+                            <li><a href="#">Contact</a></li>
+                        </ul>
+                    </div>
+                    <div class="menu2 flex-spaceBetween">
+                        <ul class="menu-item2">
+                            <li><a href="#">LATEST</a></li>
+                            <li><a href="#">HARD FORK</a></li>
+                            <li><a href="#">PLUGGED</a></li>
+                            <li><a href="#">README</a></li>
+                            <li><a href="#">GROWTH QUARTERS</a></li>
+                            <li><a href="#">NEURAL</a></li>
+                        </ul>
+                        <ul>
+                            <li><i class="fas fa-search"></i></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>    
+
+            <div class="menuMobile">
+                <div class="mobileMenu flex-spaceBetween">
+                    <div><img src="assets/images/logo2.PNG" alt="logo-image"></div>
+                    <div>
+                        <i class="fas fa-search"></i>
+                        <i class="fas fa-bars"></i>
+                    </div>
+                </div>
+                <div class="weHeard">
+                    We heard you like events and sweatpants so we made event you
+                    can join in your seatpants <i class="fas fa-long-arrow-alt-right"></i>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <section class="news-section flex">
+        <div class="news__article1">
+            <div class="feature-main__top">
+                <img src="assets/images/tnw.svg" alt="" class="feature-article__logo" />
+            </div>
+            <div class="title__bottom avalon">
+                <h3 class="news__title1">
+                    US Techs execs have dumped $3.6 billion in company stock this month
+                </h3>
+                <div class="news__author">
```

Comment:
This and all div elements with this class -> `.news__author` changes position when height changes in small screen mode.